packages/nocodb/src/lib/migrations/v2/nc_018_add_download_in_view.ts

```diff
@@ -0,0 +1,38 @@
+import Knex from 'knex';
+import { MetaTable } from '../../utils/globals';
+
+const up = async (knex: Knex) => {
+  await knex.schema.alterTable(MetaTable.VIEWS, (table) => {
+    table.boolean('download').defaultTo(true);
```

Comment:
What about keeping a meta(`text` type) column that holds a JSON string?  Extending view-specific metadata becomes more manageable that way we don't have to write more migrations.